packages/gatsby-theme-store/src/sdk/orderForm/useLogger.tsx

```diff
@@ -0,0 +1,7 @@
+import { useCallback } from 'react'
+
+export const useLogger = () => {
+  const log = useCallback(() => {}, [])
```

Comment:
what is it? 🤔

Reply:
Nope, this function is required, for now, that we don't have a logger we need to fake it for the package.